tests/plugins/device.py

```diff
@@ -228,7 +228,10 @@ def device_get_imu_sample(
             assert self._opened is True
             if not self._cameras_started:
                 return Result.Failed.value, None
-            return Result.Success.value, (36.6, (0.1, 9.8, 0.005), time.time_ns(), (0.1, 0.2, 0.3), time.time_ns())
+            return (
+                Result.Success.value,
+                (36.6, (0.1, 9.8, 0.005), int(time.time() * 1000), (0.1, 0.2, 0.3), int(time.time() * 1000)),
+            )
```

Comment:
time.time_ns() appeared in 3.7.

Reply:
I might be wrong, but is it `time.time()` in second and it should be multiply by 1000^2 to be in ns?

Reply:
s*1e9 == ns

Reply:
Actually the imu timestamp is in us so it should be 1e6
https://microsoft.github.io/Azure-Kinect-Sensor-SDK/master/structk4a__imu__sample__t.html